### PR TITLE
release: update manifests and helm chart for v0.0.22

### PR DESCRIPTION
**Pull request by tam7t**
Signed-off-by: Tommy Murphy <tommymurphy@google.com>

**What this PR does / why we need it**:

v0.0.22 release

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/secrets-store-csi-driver/README.md [hunk 1]
```diff
@@ -19,67 +19,73 @@ $ helm install csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driv
...
+| `linux.tolerations`                     | Tolerations for the daemonset on linux nodes                                                                                      | `[]`                                                    |
+| `linux.metricsAddr`                     | The address the metric endpoint binds to                                                                                          | `:8095`                                                 |
+| `linux.registrarImage.repository`       | Linux node-driver-registrar image repository                                                                                      | `k8s.gcr.io/sig-storage/csi-node-driver-registrar`      |
+| `linux.registrarImage.pullPolicy`       | Linux node-driver-registrar image pull policy                                                                                     | `IfNotPresent`                                          |
+| `linux.registrarImage.tag`              | Linux node-driver-registrar image tag                                                                                             | `v2.1.0`                                                |
```
Comment (aramase) [inline charts/secrets-store-csi-driver/README.md:39]: Could you also update this image to `v2.2.0` for linux and windows in `charts` and `manifest_staging/charts`?

Comment (aramase) [reply charts/secrets-store-csi-driver/README.md:39]: For follow up we should update the manifest_staging readme with the new image versions when we update the images in the `values.yaml`.

File: charts/secrets-store-csi-driver/README.md [hunk 2]
```diff
@@ -19,67 +19,73 @@ $ helm install csi-secrets-store secrets-store-csi-driver/secrets-store-csi-driv
...
+| `windows.tolerations`                   | Tolerations for the daemonset on windows nodes                                                                                    | `[]`                                                    |
+| `windows.metricsAddr`                   | The address the metric endpoint binds to                                                                                          | `:8095`                                                 |
+| `windows.registrarImage.repository`     | Windows node-driver-registrar image repository                                                                                    | `k8s.gcr.io/sig-storage/csi-node-driver-registrar`      |
+| `windows.registrarImage.pullPolicy`     | Windows node-driver-registrar image pull policy                                                                                   | `IfNotPresent`                                          |
+| `windows.registrarImage.tag`            | Windows node-driver-registrar image tag                                                                                           | `v2.1.0`                                                |
```
Comment (aramase) [inline charts/secrets-store-csi-driver/README.md:65]: ditto

File: manifest_staging/charts/secrets-store-csi-driver/templates/role.yaml [hunk 3]
```diff
@@ -50,14 +50,4 @@ rules:
   - get
   - patch
   - update
-{{- if .Values.rbac.pspEnabled }}
```
Comment (aramase) [inline manifest_staging/charts/secrets-store-csi-driver/templates/role.yaml:53]: I think `make manifests` is removing the psp section. We'll need to add this manually to both `charts/` and `manifest_staging/charts` and then add it as part of kustomize. WDYT?

File: charts/secrets-store-csi-driver/README.md (hunk 1 above)
Comment (tam7t) [reply charts/secrets-store-csi-driver/README.md:39]: Ah so this was built off of `manifest_staging/charts/secrets-store-csi-driver/README.md`, and we just forgot to update the values on the PRs that updated `manifest_staging...values.yaml`?

Comment (aramase) [reply charts/secrets-store-csi-driver/README.md:39]: That's right!

**Review by aramase — APPROVED** [review]
/lgtm

**Pull request merged**